plugins/assimp/module/vtkF3DAssimpImporter.cxx

```diff
@@ -602,7 +602,8 @@ class vtkF3DAssimpImporter::vtkInternals
     }
     catch (const DeadlyImportError& e)
     {
-      vtkWarningWithObjectMacro(this->Parent, "Assimp exception: " << e.what());
+      vtkErrorWithObjectMacro(this->Parent, "Assimp exception: " << e.what());
+      return false;
```

Comment:
Cannot be covered?

Reply:
According to Assimp, no, but I was not able to trigger that code anyway.

Reply:
I tried modifying the fbx binary manually but always hit other errors, never this one, even when trying to read assimp code.